src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -95,6 +98,11 @@ public Long submitAppeal(SyaCaseWrapper appeal, String userToken) {
         appeal.setCaseType("draft");
 
         IdamTokens idamTokens = getUserTokens(oauth2Token);
+        if (hasValidCaseworkerRole(idamTokens)) {
```

Comment:
This same block is repeated multiple times

Reply:
I also think 'hasValidCaseworkerRole' is a confusing name. You don't expect an exception to be thrown if something is valid. Something like "isCaseWorker" reads better.